src/components/Chart/LineChart.jsx

```diff
@@ -11,10 +11,19 @@ class LineChart extends Component {
     this.updateData()
   }
 
+  getRootWidth() {
+    if (typeof this.props.width === 'number') {
+      return this.props.width
+    }
+
+    const { width } = getComputedStyle(this.root)
+
+    return +width.slice(0, -2)
```

Comment:
```parseInt(width)``` would also work and would be clearer IMH

```
> parseInt('12px')
12
```

Reply:
TIL `parseInt` stops parsing when it finds a character that is not a number :+1: